src/platforms/rcore_comma.c

```diff
@@ -647,6 +649,9 @@ void PollInputEvents(void) {
           }
 
         } else if (platform.touch.fingers[i].action == TOUCH_ACTION_UP) {
+          if (CORE.Input.Touch.currentTouchState[i] == 1) {
+            released_detected = 1;
+          }
```

Comment:
this if statement is redundant right? if it happened in the same frame, current would get set to 1 from that. if it happened in previous frames (normal), current touch state would also still be 1?